histolab/masks.py

```diff
@@ -89,8 +100,14 @@ def _mask(self, slide) -> np.ndarray:
             The dimensions are those of the thumbnail.
         """
         thumb = slide.thumbnail
-        filters = FiltersComposition(histolab.slide.Slide).tissue_mask_filters
-        thumb_mask = filters(thumb)
+
+        # Generate appropriate composition of filter
```

Comment:
This comment IMO is not meaningful. We should expand it explaining each `if` branch or get rid of it